web-registry/src/lib/clients/transferQueryClient.ts

```diff
@@ -0,0 +1,9 @@
+import { QueryClientImpl } from '@regen-network/api/lib/generated/ibc/applications/transfer/v1/query';
+
+import { connect as connectToApi } from '../../ledger';
+
+export const getTransferQueryClient = async (): Promise<QueryClientImpl> => {
+  const api = await connectToApi();
+  if (!api || !api?.queryClient) return Promise.reject();
+  return new QueryClientImpl(api.queryClient);
+};
```

Comment:
maybe it would be clearer to call this client `ibcQueryClient` and `getIbcQueryClient` ?

Reply:
`ibcQueryClient` won't work if we start using different clients from `ibc`, `ibcTransferQueryClient` could work it's just a bit longer. If we go with this naming convention, we will have other clients like `getEcocreditMarketplaceClient`. 
If we are fine with the length we can go with this.

Reply:
mm, good point... we can leave it like that for now and discuss it, but if we are going to have more clients, perhaps we are interested in having the most complete names to identify them more easily

Reply:
Yes, I guess it's a trade-off between readability and import uniqueness. Personally, I would prefer the shorter version because figuring out the good import happens only once while using the shorter version helps with readability every time we open the file.

Reply:
agreed it's ok for now but I guess in the future, we could just use subfolders instead of long names to mimic the proto generated file structures, eg:
`clients/regen/ecocredit/queryClient.ts`
`clients/regen/ecocredit/marketplace/queryClient.ts`
`clients/ibc/applications/transfer/queryClient.ts`